src/dependencies/dss-flappers/FlapperInstance.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.8.0;
+
+struct FlapperInstance {
+    address flapper;
+    address mom;
+}
```

Comment:
Please add line to the end of the file, [if you need reasons why](https://unix.stackexchange.com/a/18789)

Also, why was this moved to a separate file?

Reply:
Done, it was an oversight when copying and pasting the content.
To replicate exactly how it is the audited code: https://github.com/makerdao/dss-flappers/tree/master/deploy